jaeger-tracerresolver/README.md

```diff
@@ -0,0 +1,38 @@
+# Jaeger Tracer Resolver
+
+This module provides a Jaeger implementation for the [TracerResolver](https://github.com/opentracing-contrib/java-tracerresolver). This mechanism provides a vendor neutral approach for obtaining a `Tracer` using the JDK
+`ServiceLoader`.
+
+
+## Maven Dependency
+```xml
+<dependency>
+    <groupId>com.uber.jaeger</groupId>
+    <artifactId>jaeger-tracerresolver</artifactId>
+    <version>$jaegerVersion</version>
+</dependency>
+```
+
+## Usage
+
+```java
+Tracer tracer = TracerResolver.resolveTracer();
+```
+
+## Configuration Options
+
+The following configuration properties can be provided either as an environment variable or system property.
```

Comment:
Should we move these to the README in core as well? We can link to that file from here.

Right now the main README only suggests
```java
Configuration config = new Configuration("myServiceName", null, null);
```
for production. I would add a sub-section there "Configuration via Environment" with what we have here. And it would also make sense to add yet another subsection that it's possible to init via TracerLoader, and link to here.